handler_models.go

```diff
@@ -199,7 +197,6 @@ func ModelOwnerLikeRemove(owner, name string, user *users.User, tx *gorm.DB,
 	}
 
 	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
-	fmt.Fprint(w, count)
```

Comment:
Have you checked that removing this line doesn't create an issue in the frontend? I see the value on returning the count after removing the like to the frontend.

Reply:
You already mentioned this in the PR, resolving 😃

Reply:
Yes, I stated so in the PR description. Thanks for double checking.